src/administrator/models/forms/eventticket.xml

```diff
@@ -62,6 +62,7 @@
   "xswa": false,
   "qualification": false,
   "committee": false,
+  "first_event": false,
```

Comment:
Need to add this option to the [event_ticket.rst](https://github.com/SWAuk/com_swa/blob/master/docs/event_ticket.rst) doc. Be good if this was also exported to PDF and saved in Google Drive overwriting the [previous version](https://drive.google.com/drive/folders/0B0Bq39ToJiiWaDJleUlPZXRSWkE).